UnityProject/Assets/Scripts/MapSaver/MapSaver.cs

```diff
@@ -381,6 +381,110 @@ public bool RemoveEmptys()
 				FieldsToRefresh.Clear();
 			}
 
+			if (Cut)
+			{
+				var PresentTiles = MetaTileMap.PresentTilesNeedsLock;
+
+				List<TileLocation> TilesToRemove = new List<TileLocation>();
+
+				lock (PresentTiles)
+				{
+					foreach (var TileMaps in PresentTiles)
+					{
+						foreach (var Tile in TileMaps)
+						{
+							if (AllowedPoints == null)
+							{
+								TilesToRemove.Add(Tile);
+							}
+							else
+							{
+								if (AllowedPoints.Contains(Tile.LocalPosition))
+								{
+									TilesToRemove.Add(Tile);
+								}
+							}
+						}
+					}
+				}
+
+				var MultilayerPresentTiles = MetaTileMap.MultilayerPresentTilesNeedsLock;
+
+				lock (MultilayerPresentTiles)
+				{
+					foreach (var TileMaps in MultilayerPresentTiles)
+					{
+						foreach (var Tiles in TileMaps)
+						{
+							if (Tiles == null || Tiles.Count == 0) continue;
+							if (AllowedPoints == null)
+							{
+								TilesToRemove.AddRange(Tiles);
+							}
+							else
+							{
+
+								Vector3Int GoodPOS = Vector3Int.one;
+								bool Found = false;
+								foreach (var Tile in Tiles)
+								{
+									if (Tile != null)
+									{
+										GoodPOS = Tile.LocalPosition;
+										Found = true;
+										break;
+									}
+								}
+
+								if (Found == false) continue;
+								GoodPOS.z = 0; //TODO Tile map upgrade
+								if (AllowedPoints.Contains(GoodPOS))
+								{
+									TilesToRemove.AddRange(Tiles);
+								}
+							}
+						}
+					}
+				}
+
+
+				foreach (var Location in TilesToRemove)
+				{
+					Location?.Remove(false);
+				}
+
+
+				IEnumerable<RegisterTile> Objects =null;
+				if (Application.isPlaying)
+				{
+					Objects = MetaTileMap.ObjectLayer.GetTileList(CustomNetworkManager.Instance._isServer).AllObjects; //TODO Disabled objectsxz
+				}
+				else
+				{
+					Objects = MetaTileMap.GetComponentsInChildren<RegisterTile>(true);
+#if UNITY_EDITOR
+					CommonManagerEditorOnly.Instance.CustomNetworkManagerPrefab.ForeverIDLookupSpawnablePrefabs.Clear();
+					CommonManagerEditorOnly.Instance.CustomNetworkManagerPrefab.SetUpSpawnablePrefabsForEverIDManual();
+					CustomNetworkManager.Instance = CommonManagerEditorOnly.Instance.CustomNetworkManagerPrefab;
+#endif
+				}
+
+				foreach (var Object in Objects)
```

Comment:
This hypothetical Cut function could then be further divided into a CutRegionTiles() and a CutRegionObjects() method.

Reply:
wanted to integrated as part of the tile map stuff but I don't wan the destroyed tiles  dropping stuff messing up the save